src/main/java/com/jalasoft/jfc/model/ImageConverter.java

```diff
@@ -21,9 +21,11 @@
  * @version 0.1 11 Dec 2019.
  * @author Oscar Lopez.
  * */
-public class Convert {
-    protected String myPath = "C:\\Program Files (x86)\\ImageMagick-6.3.9-Q8\\"; //path where ImageMagick is installed.
-    private final int THUMBNAILVALUE = 128; //default value for creating thumbnail.
+public class ImageConverter {
+    //path where ImageMagick is installed.
+    protected String myPath = "C:\\Program Files (x86)\\ImageMagick-6.3.9-Q8\\";
+    //default value for creating thumbnail.
+    private final int THUMBNAILVALUE = 128;
 
     /**
      * convertImage Method por change a Image format, resize,
```

Comment:
improve this documentation there are misspelings "convertImage Method por change a Image format, resize".
for example "convertImage Method **f**or chang**ing** a**n** Image format **to another one**, **and** resize **it**"